Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+3. There are 54 digital input/output pins (of which 15 are PWM outputs) and 16 analog inputs.  This meets the needs for at least 15 of each (digital and analog input pins).  The other 24 digital pins may also be assigned special purposes, so the excess will be used.  
+4. The cpu clock speed is 16 MHz.  This will suffice for computations and startup speeds to be adequate.  
+5. The code is currently of an unknown size, so it is necessary to aim slightly higher in terms of needed processing memory.  The Arduino Mega has 256 KB of Flash, 8 KB of SRAM, and 4 KB of EEPROM.  The SRAM being bigger would be nice, but the Flash memory is satisfactory for the subsystem’s needs.
+6. Recommended voltage of 7-12 V and a current draw of 20 mA puts the Mega near the target power draw of 240 mW.  This should prove to not be too power hungry and help to meet some of the operational time specifications of the project.
+7. Algorithmic measurement of the battery life is supported.  This is as the board will be able to read the voltage levels coming from the battery and run an algorithm based on the battery’s expected output to determine the battery’s capacity.
+8. Physically, the board is slightly bigger when compared to the other boards, but the extra space to support other components.  The board has a length of 4 inches and width of 2.1 inches with less than an inch in height.  This does put it below the volume cap.
+9. Arduino is a very popular and respected manufacturing and electronics design company.  They were established in 2005 and have had a lot of support throughout the years.  It is expected for them to persist for quite some time.
+10. The Arduino Mega supports C and Assembly for certain.  It could even be said that there is a favor for C when it comes to coding Arduino boards.  Arduino does have Python support as well and can potentially be used to accomplish the goals of this project or allow for the code to be translated from C to Python later on for whatever reason may arise from this team or another.
+
+	
+Arduino has been used for many, many projects in history and has a good track record.  The Arduino Mega and USB Host Shield have conjointly shown to fulfill the needs of this subsystem.  The hardware and software specifications are being checked off in their entirety and then some for potential wiggle room later in the project.  This gives more options and improvements later or prevents the need for cutting corners due to any restrictions that would arise from having a smaller and less powerful board.  Going with another option to save on the budget could prove to be a fatal flaw later on if the board doesn’t have enough room, memory, or power to get the job done.  Going a little extra seems reasonable and advisable.
+
+## BILL OF MATERIALS (BOM)
+
+| Designator | Manufactured Part # | Quantity | Description                                                   | Price  |
```

Comment:
Add a column for Manufacturer or include that in a Manufacturer/PartNumber column. Move Quantity to be next to the price column. Label price column as "Price Each" to eliminate ambiguity.